src/pal/src/misc/fmtmessage.cpp

```diff
@@ -51,11 +51,15 @@ typedef VOID * HSATELLITE;
 static HSATELLITE s_hSatellite = NULL;
 static LPVOID s_lpLibRotorPalRt = NULL;
 typedef HSATELLITE (__stdcall *FnLoadSatelliteResource)(LPCWSTR);
-static FnLoadSatelliteResource LoadSatelliteResource;
 typedef BOOL (__stdcall *FnFreeSatelliteResource)(HSATELLITE);
 static FnFreeSatelliteResource FreeSatelliteResource;
+#ifndef FEATURE_PAL
```

Comment:
Ah right, when I was modifying the file, for some weird reason I have not realized it is in PAL when I was making the change. Ok, I'll use #if 0 instead. I'd like to preserve the old code until the final solution is implemented.